pantos/client/library/blockchains/base.py

```diff
@@ -20,6 +23,13 @@
 from pantos.client.library.configuration import get_blockchain_config
 from pantos.client.library.exceptions import ClientLibraryError
 
+_CONTRACTS_VERSION = semantic_version.Version('1.0.0')
```

Comment:
The current version is 1.1.2

Reply:
The ABIs have a separate version, we only have 1.0.0 so far:
https://github.com/levonyak/pantos-common/tree/main/pantos/common/blockchains/contracts